cloud

```diff
@@ -1243,6 +1244,23 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+# Deploy a Google Cloud Storage Bucket
+# Args: project bucket
+function storage_deploy() {
+    declare -r project="$1"; shift
+    declare -r bucket="$1"; shift
+    TMPDIR="$TMPDIR_ORIG" gsutil -q mb -p "$project" -c STANDARD -l \
+    "us-central1" -b on "gs://$bucket"
```

Comment:
Please add one more indent level to this line, so it's clearer it's continuing from the previous one, as is done for the rest of the file. It's easy to miss just the backslash otherwise.

Reply:
done